### DISCUSSION TITLE
DataGrid DropDown From list

### DISCUSSION THREAD

**Discussion by peterhym21** [Megabit/Blazorise#3782] [Q&A] [answered]
What i want to do is when i am editing my datagrid, I want a dropdown that gets data for a List

                               <DataGrid TItem="UiOperationListVm"
                                  Data="@Operations"
                                  @bind-SelectedRow="@SelectedOperation"
                                  EditMode="DataGridEditMode.Inline"
                                  Editable
                                  PageSize="10"
                                  ShowPager
                                  Responsive>
                                <DataGridColumn Field="@nameof(UiOperationListVm.Id)" Caption="ID" Sortable="false" />
                                <DataGridColumn Field="@nameof(UiOperationListVm.Name)" Caption="Name" Editable />
                                <DataGridColumn Field="@nameof(UiOperationListVm.Description)" Caption="Description" Editable />
                                <DataGridColumn Caption="Operation types">
                               
                                // Dropdown for List
                               
                                </DataGridColumn>
                            <DataGridCommandColumn>
                                <EditCommandTemplate>
                                    <Button Color="Color.Primary" Clicked="@context.Clicked">Edit</Button>
                                </EditCommandTemplate>
                                <DeleteCommandTemplate></DeleteCommandTemplate>
                                <SaveCommandTemplate>
                                    <Button Color="Color.Primary"                                                                                                                                                            Clicked="@(()=>OnSaveClickedOperation(context))">@context.LocalizationString</Button>
                                </SaveCommandTemplate>
                                <CancelCommandTemplate><Button Color="Color.Danger" Clicked="@context.Clicked">Cancel</Button></CancelCommandTemplate>
                            </DataGridCommandColumn>
                        </DataGrid>

**Comment by stsrki** [comment, marked as answer]
You can use DisplayTemplate.

Example:

```
<DataGridColumn Caption="Operation types">
  <DisplayTemplate>
    // add dropdown here. Also you can access the item by context.Item
  </DisplayTemplate>
</DataGridColumn>
```

**Reply by peterhym21** [reply]
thank you so much for the help and fast reply
really apricate it :)
